Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -117,9 +124,11 @@ public extension CBPurchase {
         self.purchaseProductHandler(product: product, completion: handler)
     }
     
-    //Restore the purchase
-    func restorePurchases(completion handler: @escaping ((_ result: Result<(status:Bool, subscriptionId:String?, planId:String?), Error>) -> Void)) {
-        buyProductHandler = handler
+    func restorePurchases(includeNonActiveProducts: Bool? = false,completion handler: @escaping ((_ result: Result<[InAppSubscription], RestoreError>) -> Void)) {
```

Comment:
This needn't be optional if we are providing the default value.

Reply:
updated